Form-Controls/index.html

```diff
@@ -9,10 +9,35 @@
   </head>
   <body>
     <header>
-      <h1>Product Pick</h1>
+      <h1>Please choose your T-Shirt</h1>
     </header>
     <main>
       <form>
+       <p> <span style="color:red">*</span><span><label for="FullName">Name: </label><input type="text" id="FullName" name="FullName" required pattern="[A-Za-z]{3,}"></span></p>
```

Comment:
Please, avoid inline styling and use separate file (style.css) or at least add all your styles into `<style>` section in your HTML.

Reply:
For attributes, please, use lowercase values and hyphens.

Bad:
for="FullName"

Good:
for="full-name"